wgpu-hal/src/metal/adapter.rs

```diff
@@ -232,6 +232,7 @@ impl crate::Adapter<super::Api> for super::Adapter {
                 }
                 flags
             }
+            Tf::NV12 => unreachable!(),
```

Comment:
This doesn't look unreachable to me. Would it make sense to return empty caps if we don't handle this format in metal?